examples/rl_sim_cpp/azure_credentials.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <memory>
+
+#include "api_status.h"
+#include "configuration.h"
+
+#include "future_compat.h"
+
+#include <azure/identity/default_azure_credential.hpp>
+#include <azure/identity/azure_cli_credential.hpp>
+
+#include <chrono>
+#include <string>
+
+class AzureCredentials
+{
+public:
+  AzureCredentials();
+  int get_credentials(std::string& token_out, std::chrono::system_clock::time_point& expiry_out,
```

Comment:
Out params should be last

Reply:
Thats our standard here? I'll get that fixed

Reply:
I thought it was a pretty common standard, not sure if we have a convention or not in this codebase